docs/src/piccolo/schema/constraints.rst

```diff
@@ -0,0 +1,34 @@
+===========
+Constraints
+===========
+
+Unique constraints
+==================
+
+Single column
+-------------
+
+Unique constraints can be added to a single column using the ``unique=True``
+argument of ``Column``:
+
+.. code-block:: python
+
+    class Band(Table):
+        name = Varchar(unique=True)
+
+Multi-column
+------------
+
+Use the ``add_unique_constraint`` method to add a multi-column constraint to a
+``Table``:
+
+.. currentmodule:: piccolo.table
+
+.. automethod:: Table.add_unique_constraint
+
+-------------------------------------------------------------------------------
+
+Check constraints
+=================
+
+.. automethod:: Table.add_check_constraint
```

Comment:
@dantownsend For now, constraints are droped by simply deleting (or commenting out) the constraints from the table definition. I'm fine with that, but it should be indicated in the docs so the user knows how to use that. Another option is to create a `drop_unique_constraint` classmethod that will drop the constraints. As you wish, both options suit me.